apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -880,6 +903,7 @@ wrong_sig_create({I, R, #{initiator_amount := IAmt0, responder_amount := RAmt0,
 
 check_incorrect_deposit(Cfg) ->
     Debug = true,
+    Fun = proplists:get_value(wrong_action, Cfg),
```

Comment:
In cases such as these, I personally prefer `{_, Fun} = lists:keyfind(wrong_action, 1, Cfg)`, although that also fails to verify that `Fun` is, in fact, a `fun/4`.